wiki/Ranking_Criteria/ru.md

```diff
@@ -1,5 +1,6 @@
 ---
 outdated: true
+outdated_since: 7742320aaa249169bcb0536d8fc19dc7d32015fe
```

Comment:
uhh this is not true (it's already outdated since we don't know when, so you can't add `outdated_since`)

Reply:
that particular one wasn't already marked as outdated, so I assumed it was up to date prior to my change